src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -7,16 +7,59 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
 /**
- * Responds with a hard-coded message for testing purposes.
+ * Handles fetching and saving user data.
  */
 @WebServlet("/about")
-public class AboutMeServlet extends HttpServlet{
+public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+ @Override
```

Comment:
Style: indentation for Java should be 2 spaces

Reply:
I have change that.